src/components/Calculator.js

```diff
@@ -1,38 +1,74 @@
 /* eslint-disable */
 import React from 'react';
+import calculate from '../logic/calculate';
 import './Calculator.css';
 
```

Comment:
- [ ] OPTIONAL
Performing a calculation yields correct output, however, your display needs some improvements. When enter an operation, for instance `9 - 2`, your display reverses it to `2 - 9`. Kindly ensure that your display is outputting the correct operations entered.

Reply:
@rloterh got it thanks a lot for mentioning that I already fixed the problem